packages/adminrouter/extra/src/test-harness/modules/generic_test_code/common.py

```diff
@@ -97,6 +98,67 @@ def generic_response_headers_verify_test(
             header_is_absent(resp.headers.items(), name)
 
 
+def generic_upstream_cookies_verify_test(
+        ar,
+        headers,
+        path,
+        cookies_to_send=None,
+        assert_cookies_present=None,
+        assert_cookies_absent=None):
+    """Test if cookies that are passed to the upstream by AR are correct
+
+    Helper function meant to simplify writing multiple tests testing the
+    same thing for different endpoints.
+
+    Arguments:
+        ar: Admin Router object, an instance of runner.(ee|open).Nginx
+        headers (dict): headers dict that contains JWT and cookies. The auth
```

Comment:
Please replace "JWT" with what you actually mean. It's probably "the DC/OS authentication token".